conductor_api/src/config.rs

```diff
@@ -120,7 +120,7 @@ pub fn default_persistence_dir() -> PathBuf {
 #[derive(Deserialize, Serialize, Clone, Debug)]
```

Comment:
This rust-doc comments seems out of date. Can we get an updated description?

Reply:
Good catch as usual ! Thanks !